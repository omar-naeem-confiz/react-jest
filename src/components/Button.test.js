import React from 'react';
import { mount } from 'enzyme';
import { Button } from '.';

it('render Button', () => {
  const wrapper = mount(
    <Button
      label="Add"
      onClick={() => {
        expect(1).toBe(1)
      }}
    />
  );
  expect(wrapper).toMatchSnapshot();
  wrapper.find('button').simulate('click')
});
