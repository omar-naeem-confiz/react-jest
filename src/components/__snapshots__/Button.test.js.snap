// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Button 1`] = `
<Button
  label="Add"
  onClick={[Function]}
>
  <button
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#017acd",
        "borderRadius": 6,
        "height": 54,
        "minWidth": 127,
        "padding": 10,
        "paddingLeft": 40,
        "paddingRight": 40,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 16,
          "fontWeight": "bold",
        }
      }
    >
      <View
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "fontWeight": "bold",
          }
        }
      >
        <div
          style={
            Object {
              "color": "white",
              "display": "flex",
              "flexDirection": "column",
              "fontSize": 16,
              "fontWeight": "bold",
            }
          }
        >
          Add
        </div>
      </View>
    </Text>
  </button>
</Button>
`;
